tst/testinstall/compressed.tst

```diff
@@ -0,0 +1,169 @@
+#@local dir,fname,isGzippedFile,stream,str
+gap> START_TEST("compressed.tst");
+gap> dir := DirectoryTemporary();;
+gap> fname := Filename(dir, "test.g.gz");;
+
+# Let us check when we have written a compressed file by checking the gzip header
+gap> isGzippedFile := function(dir, name)
+>    local out, str,prog;
+>    str := "";
+>    out := OutputTextString(str, true);
+>    Process(dir, Filename(DirectoriesSystemPrograms(),"cat"), InputTextNone(), out, [name]);
+>    return str{[1..2]} = "\037\213";
+>  end;;
+gap> str := "hello\ngoodbye\n";;
+
+# Write a compressed file
+gap> FileString( Filename(dir, "test.g.gz"), str ) = Length(str);
+true
+
+# Check file really is compressed
+gap> isGzippedFile(dir, "test.g.gz");
+true
+
+# Check reading compressed file
+gap> StringFile( Filename(dir, "test.g.gz") ) = str;
+true
+
+# Check gz is added transparently
+gap> StringFile( Filename(dir, "test.g") ) = str;
```

Comment:
```suggestion
gap> StringFile( fname ) = str;
```

Reply:
This one isn't the same (doesn't have the .gz)